CMakeLists.txt

```diff
@@ -26,6 +23,13 @@ option(DEFLATE_FREESTANDING "build a freestanding library, i.e. a library that d
 option(DEFLATE_BUILD_TEST_PROGRAMS "Build test programs" OFF)
 option(DEFLATE_TEST_SUPPORT__DO_NOT_USE "Don't use this option except for testing; it isn't a stable interface." OFF)
 
```

Comment:
The options are prefixed with DEFLATE, while the library project is now libdeflate. Should we make that consistent ?

Reply:
Indeed, it would be more consistent